### ISSUE TITLE
How to add a new dataset?

### ISSUE THREAD

**Issue by DongdongY1** [jzbjyb/FLARE#4]
I would like to try a new dataset in place of the wiki dump. It seems that I need to compress documents into .tsv format and make some adjustments over code. Could some simple instructions be given?

**Comment by flatLying** [comment]
yep, could please release some instruction about how to replace wiki data with our own dataset
